packages/api/src/store/db.ts

```diff
@@ -243,4 +257,23 @@ async function ensureDatabase(postgresUrl) {
   adminPool.end();
 }
 
+async function createCdnUsageRegTable(db) {
+  await db.query(`
+          CREATE TABLE IF NOT EXISTS cdn_usage_reg (
+            date timestamp without time zone NOT NULL,
+            region character varying(128) COLLATE pg_catalog."default" NOT NULL,
+            playback_id character varying(128) COLLATE pg_catalog."default" NOT NULL,
+            user_id character varying(128) COLLATE pg_catalog."default" NOT NULL,
+            user_email character varying(512) COLLATE pg_catalog."default" NOT NULL,
+            unique_users integer NOT NULL,
+            total_filesize bigint NOT NULL,
+            total_cs_bytes bigint NOT NULL,
+            total_sc_bytes bigint NOT NULL,
+            count integer NOT NULL,
+            CONSTRAINT prim PRIMARY KEY (date, region, playback_id)
```

Comment:
I think it _might_ be better to have a single separate column for the `PRIMARY KEY`, like an auto-inremented `id` on the simplest case or a manually generated UUID like we have for our other tables (bonus of consistency on that).

Not sure if that is the case for Postgres, but I heard that at least for MySQL it's much more performant to keep the primary key as simple as possible for low-level data access in the storage system, and create separate more complex indexes for any other helpful queries we may want to run.

So here maybe have an `id VARCHAR(128)` (UUID) column as the primary key and then create helpful indexes for querying. This current might be one of them, and I believe we might want one with the `user_id` as well. Maybe just add the `user_id` before the `playback_id`, since any query for a `playback_id` could/should always filter by the `user_id` as well? `(date, region, user_id, playback_id)`